spec/student_spec.rb

```diff
@@ -29,7 +32,8 @@
   end
 
   describe "#create_table" do
-    it 'creates the students table in the database' do
+    it 'creates the students table in the database', :skip_before do
+      DB[:conn].execute("DROP TABLE students")
```

Comment:
Thanks for changing this! We should still have "IF EXISTS" as part of the drop, right?